internal/store/db/redis/models/storedobject.go

```diff
@@ -43,18 +43,22 @@ type StoredObject struct {
 
 	// CorrelationID is an identifier provided by EdgeX to track this record as it moves
 	CorrelationID string `json:"correlationID"`
+
+	// ContextData is a snapshot of data used by the pipeline at runtime
+	ContextData map[string]string
 }
 
 // ToContract builds a contract out of the supplied model.
 func (o StoredObject) ToContract() contracts.StoredObject {
 	return contracts.StoredObject{
 		ID:               o.ID,
-		AppServiceKey:    o.AppServiceKey,
+		AppServiceKey:    o.AppServiceKey, // ContextData is additional
```

Comment:
What is the purpose of this comment?

Reply:
Ummmm, stray artifact from IDE formatting maybe?